LA notes.tex

```diff
@@ -622,6 +617,10 @@ \section{Solution Sets of Linear Systems}\label{s1}
 What is the reduced row echelon form of $\begin{bmatrix} -3&-1&1 \\1&2&3 \\2&5&8\end{bmatrix}$? You should use the information given above and \textbf{not} a lot of calculations.
 \eq
 
+\begin{theorem} The pivot columns of a matrix $A$ generate $Col(A)$. This means that if $\vec{v} \in Col(A)$, then $\vec{v}$ can be written as a linear combination using only the pivot columns of $A$.
+\end{theorem}
+Note that this theorem uses the pivot columns of $A$ and \underline{\textbf{not}} the pivot columns of the \emph{echelon form} of $A$. Even though you need the echelon form to figure out which columns have pivots, you should use the appropriate columns of $A$ in your linear combination.
+
```

Comment:
I thought the students should practice column space examples before this theorem appeared, so they could understand its significance.